Makefile

```diff
@@ -36,7 +36,7 @@ $(BASE): ; $(info  setting GOPATH...)
 
 GOLINT = $(GOBIN)/golint
 $(GOBIN)/golint: | $(BASE) ; $(info  building golint...)
-	$Q go get -u golang.org/x/lint/golint
+	$Q go install -mod=mod golang.org/x/lint/golint
 	# golint installation modifies go.mod and it causes golint failure so run mod tidy here
```

Comment:
Do I get it correctly, that when version is specified, go.mod is not updated, and thus go mod tidy is not needed?
In that case, we could remove the comment and the go mod tidy command

Reply:
Let me try